menas/src/main/scala/za/co/absa/enceladus/menas/auth/jwt/JwtAuthenticationFilter.scala

```diff
@@ -74,8 +74,9 @@ class JwtAuthenticationFilter @Autowired()(jwtFactory: JwtFactory) extends OnceP
   }
 
   private def parseJwtAuthorities(jwtClaims: Claims): util.List[SimpleGrantedAuthority] = {
-    val groups = Option(jwtClaims.get(RolesKey, classOf[util.List[String]]))
-      .getOrElse(new util.ArrayList[String]()).asScala
+    val groups: Seq[String] = Option(jwtClaims.get(RolesKey, classOf[util.List[String]]))
+      .getOrElse(new util.ArrayList[String]())
+      .asScala
```

Comment:
```suggestion
    val groups: Seq[String] = Option(jwtClaims.get(RolesKey, classOf[util.List[String]]))
    .fold(Seq.empty[String])(_.asScala.toSeq)
```

I'd personally write it like this so that I don't have to create a new `utils.ArrayList` instance, but the more I think about it, the more I am convinced that my comment here adds little value as the _java scala interop with null_ is bound to be a bit awkward anyhow 😞